src/ui/App/App.tsx

```diff
@@ -389,6 +393,35 @@ dayjs.extend(relativeTime);
 registerPersistentRoute('/send-form');
 registerPersistentRoute('/swap-form');
 
+function GlobalKeyboardShortcuts() {
+  return (
+    <>
+      {templateData.windowContext === 'dialog' ? (
+        <KeyboardShortcut
+          combination="esc"
+          onKeyDown={() => {
+            const searchParams = new URLSearchParams(window.location.hash);
+            const windowId = searchParams.get('windowId');
+            if (windowId) {
+              windowPort.reject(windowId);
+            }
+          }}
+        />
+      ) : null}
+
+      <KeyboardShortcut
+        combination="ctrl+alt+0"
+        onKeyDown={() => {
+          // Helper for development and debugging :)
+          const url = new URL(window.location.href);
+          url.searchParams.set('windowContext', 'tab');
+          window.open(url, '_blank');
```

Comment:
While we are refactoring this part, let's change this to `openInFullScreen` (for Arc support)

Reply:
"in tab view" you mean? ;)